cabal-install/cabal-install.cabal.dev

```diff
@@ -351,7 +351,7 @@ Test-Suite unit-tests
     UnitTests.Distribution.Client.Get
     UnitTests.Distribution.Client.Glob
     UnitTests.Distribution.Client.GZipUtils
-    UnitTests.Distribution.Client.Init.FileCreators
+    UnitTests.Distribution.Client.Init
```

Comment:
This is a generated file. This modification should be done in the .zinza file, then the .dev file should be generated from that (there are instructions at the beginning of this file)

Reply:
Ah perfect. Okay